modules/apps/document-library/document-library-item-selector-web/src/main/java/com/liferay/document/library/item/selector/web/internal/util/DLBreadcrumbUtil.java

```diff
@@ -143,7 +148,17 @@ private static String _getRootFolderName(
 		Group group = themeDisplay.getScopeGroup();
 
 		if (repositoryId != 0) {
-			group = GroupServiceUtil.getGroup(repositoryId);
+			try {
+				group = GroupServiceUtil.getGroup(repositoryId);
```

Comment:
Why don't use `fetchGroup` here? fetch methods return `null` the there's no value. That way you can remove the exception handling code.

Reply:
What do you think about accessing a **local service** via a Util?

Reply:
You're right :facepalm: